.eslintrc

```
@@ -76,6 +76,23 @@
     ],
     "no-prototype-builtins": 0, // not needed as we never mess with prototype
     "no-nested-ternary": 0,
+    "no-restricted-imports": ["warn", {
+      "paths": [{
+        "name": "classnames",
+        "importNames": ["classNames"],
+        "message": "Please use classnames from classnames instead."
+      },
+      {
+        "name": "classnames",
+        "importNames": ["cx"],
+        "message": "Please use classnames from classnames instead."
+      },
+      {
+        "name": "classnames",
+        "importNames": ["cn"],
+        "message": "Please use classnames from classnames instead."
+      }]
+    }],
```

Does this close #1325 as well? I think we can change the level to `error`
```suggestion
    "no-restricted-imports": ["error", {
      "paths": [{
        "name": "classnames",
        "importNames": ["classNames"],
        "message": "Please import classnames as 'classnames'."
      },
      {
        "name": "classnames",
        "importNames": ["cx"],
        "message": "Please import classnames as 'classnames'."
      },
      {
        "name": "classnames",
        "importNames": ["cn"],
        "message": "Please import classnames as 'classnames'."
      }]
    }],
```

---

@tay1orjones That rule doesn't work because it doesn't trigger any error messages when the importNames property is used. :(

---

👍  cool. lets contain the eslint exploration in a separate PR tied to #1325